plugin-server/src/worker/vm/utils.ts

```diff
@@ -10,11 +10,12 @@ export const postgresIncrement = async (
     key: string,
     incrementBy = 1
 ): Promise<number> => {
+    // The timestamp is key to triggering the ON CONFLICT CLAUSE
     const incrementResult = await db.postgresQuery(
         `
-        INSERT INTO posthog_pluginstorage (plugin_config_id, key, value)
-        VALUES ($1, $2, $3)
-        ON CONFLICT ("plugin_config_id", "key")
+        INSERT INTO posthog_pluginstorage (plugin_config_id, key, value, timestamp)
+        VALUES ($1, $2, $3, to_timestamp(100000000000))
```

Comment:
What is the relevance of 100000000000? I can see it causing operational confusion when we see a random-looking date after rows in the db.

Unix epoch might be a smarter choise here as it might clue the investigator to see where timestamp might be equal to 0.

Reply:
In theory this could be 0 (my initial approach) but I chose a high number so that it makes sure we get this value even if for some reason `set` is used later with the same key